robottelo/config/validators.py

```diff
@@ -1,10 +1,75 @@
+        Validator(
+            'container_repo.registries.redhat.password',
+            default=secrets.get('container_repo_rh_registry_token'),
+        ),
+        Validator(
+            'container_repo.registries.quay.password',
+            default=secrets.get('container_repo_quay_robot_token'),
+        ),
+        Validator('docker.private_registry_password', default=secrets.get('satqe_shared_password')),
+        Validator('ec2.access_key', default=secrets.get('ec2_access_key')),
+        Validator('ec2.secret_key', default=secrets.get('ec2_secret_key')),
+        Validator('gce.cert', default=secrets.get('gce_cert')),
+        Validator('git.password', default=secrets.get('gitea_user_password')),
+        Validator('git.github_token', default=secrets.get('github_read_token')),
+        Validator('http_proxy.password', default=secrets.get('satqe_shared_password')),
+        Validator('ipa.idm_password', default=secrets.get('idm_password')),
+        Validator('ipa.password', default=secrets.get('ipa_password')),
+        Validator('ipa.time_based_secret', default=secrets.get('totp_ipa_secret')),
+        Validator('ldap.password', default=secrets.get('ad_password')),
+        Validator('open_ldap.password', default=secrets.get('satqe_shared_password')),
+        Validator('osp.password', default=secrets.get('osp_password')),
+        Validator('osp.project_domain_id', default=secrets.get('osp_project_domain_id')),
+        Validator('report_portal.api_key', default=secrets.get('reportportal-robottelo-token')),
+        Validator('rh_cloud.token', default=secrets.get('rh-cloud-token')),
+        Validator('rhev.password', default=secrets.get('satqe_shared_password')),
+        Validator('rhev.ca_cert', default=secrets.get('rhev_cacert')),
+        Validator('rhev.image_password', default=secrets.get('satqe_shared_password')),
+        Validator('rhsso.rhsso_password', default=secrets.get('satqe_shared_password')),
+        Validator('rhsso.totp_secret', default=secrets.get('totp_rhsso_secret')),
+        Validator('remotedb.db_password', default=secrets.get('external_db_password')),
+        Validator(
+            'virtwho.esx.hypervisor_password', default=secrets.get('esx_hypervisor_password')
+        ),
+        Validator(
+            'virtwho.xen.hypervisor_password', default=secrets.get('xen_hypervisor_password')
+        ),
+        Validator(
+            'virtwho.hyperv.hypervisor_password', default=secrets.get('hyperv_hypervisor_password')
+        ),
+        Validator(
+            'virtwho.rhevm.hypervisor_password', default=secrets.get('rhevm_hypervisor_password')
+        ),
+        Validator(
+            'virtwho.libvirt.hypervisor_password',
+            default=secrets.get('libvirt_hypervisor_password'),
+        ),
+        Validator(
+            'virtwho.kubevirt.hypervisor_password',
+            default=secrets.get('kubevirt_hypervisor_password'),
+        ),
```

Comment:
@JacobCallahan @mshriver I think we should remove these now ?

Reply:
There are still references to these secrets within the test framework, can't remove them until we actually remove those tests, which won't be happening anytime soon.